### ISSUE TITLE
RobiniaDocs API Explorer

### ISSUE THREAD

**Issue by NeuroXiq** [Testably/Testably.Abstractions#403]
Maybe You will be interested in hosting API Explorer online:

https://www.robiniadocs.com/d/testably-abstractions/api/Testably.Abstractions.Helpers.PathSystemBase.html

This is ready-to-go, so above link can be added to README.md for other people to see it.
Or using ready shields.io badge:

[![Static Badge](https://img.shields.io/badge/API%20Documentation-RobiniaDocs-43bc00?logo=readme&logoColor=white)](https://www.robiniadocs.com/d/testably-abstractions/api/Testably.Abstractions.Helpers.PathSystemBase.html
)

```
[![Static Badge](https://img.shields.io/badge/API%20Documentation-RobiniaDocs-43bc00?logo=readme&logoColor=white)](https://www.robiniadocs.com/d/testably-abstractions/api/Testably.Abstractions.Helpers.PathSystemBase.html
)
```

https://github.com/NeuroXiq/RobiniaDocs

Regards
NeuroXiq

**Comment by vbreuss** [comment]
@NeuroXiq :
Thanks for suggesting your solution for API documentation.
I had a look at it and although it looks quite neat and clear, I am missing the description for the methods and properties. That is one reason, why I don't think that this repo is a good fit for your solution, as it does not really contain an own API that needs to be documented, but is rather an abstraction for the existing .NET types. Most of the XML documentation just refers back to the .NET pendant.
For instance the [`IDateTime`](https://github.com/Testably/Testably.Abstractions/blob/main/Source/Testably.Abstractions.Interface/TimeSystem/IDateTime.cs) class contains lots of `inheritdoc` documentation to include the description from the .NET [DateTime](https://learn.microsoft.com/en-us/dotnet/api/system.datetime) class for which the official documentation is quite extensive. Your solution seems not to support this syntax, so the [generated API documentation](https://www.robiniadocs.com/d/testably-abstractions/api/Testably.Abstractions.TimeSystem.IDateTime.html) is missing a lot of information that is visible e.g. in Visual Studio with IntelliSense.

**Comment by NeuroXiq** [comment]
Thanks for response.
I checked this and You are right. Libary shows correct documentation in visual studio (I tested this by downloading package to Visual Studio) but  documents are not generated.
I tired to investigate why and my underlying tool doesn't support this kind of 'inheritdoc' tag.
So I think I can close this ticket, maybe in the futuer it will be supported :)

Buy anyway maybe You will find RobiniaDocs useful in the future or for some other projects :)

Regards 
NeuroXiq
